Lib/idlelib/config-extensions.def

```diff
@@ -19,7 +19,7 @@ max-width= 72
 style= expression
 flash-delay= 500
 bell= True
-
+highlighting = hilite
```

Comment:
There's a mismatch between "highlighting" here and "highlight" used in the config dialog.